res/circlearrow.svg

```diff
@@ -1,3 +1,6 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Comment:
I created them all by hand in an editor. Some of them I had created for the old cleopatra many years ago, using the same process.